data-raw/R2R_HRL_habitat_inputs/yuba_river_r2r_hrl_habitat.rmd

```diff
@@ -0,0 +1,156 @@
+• Accurate empirically-derived updated habitat-flow relationships would be developed after all HR&L projects are completed, modeled and evaluated. "
+•  Estimated grading extents of indivudal HR&L Projects are based on: (1) the actual grading extent for Hallwood and Lower Long Bar; and (2) estimated grading extents for Upper Long Bar and Rose Bar Comp based on preliminary or conceptual designs.
+•  Estimated proportion of in-channel and floodplain habitat of individual H&L Projects are based on: (1) As-Built DEM and hydraulic modeling of Hallwood (cbec, pers. comm.); (2) estimated based on preliminary construction information for Lower Long Bar (C. Hammersmark, pers comm.); and (3) estimated based on conceptual designs or plans of Upper Long Bar and Rose Bar Comprehensive projects.
+
+```{r, echo=FALSE}
+col_names_yuba <- read_excel("data-raw/R2R_HRL_habitat_inputs/data/HydraulicHabFlow_LYR_7-15-2024.xlsx",
+                           sheet = "LYR_HabFlowRelationships",
+                           range = "A4:K4") |> 
+  janitor::clean_names() |> 
+  names()
+
+yuba_hrl <- read_excel("data-raw/R2R_HRL_habitat_inputs/data/HydraulicHabFlow_LYR_7-15-2024.xlsx",
+                           sheet = "LYR_HabFlowRelationships",
+                           range = "A5:K40",
+                           col_names = col_names_yuba) |> 
+  select(flow_cfs, 
+         below_dpd_hrl_ic_rear = below_dpd_4, 
+         above_dpd_hrl_ic_rear = above_dpd_5,
+         below_dpd_hrl_fp_rear = below_dpd_8,
+         above_dpd_hrl_fp_rear = above_dpd_9) |> 
+  mutate(across(below_dpd_hrl_ic_rear:above_dpd_hrl_fp_rear, as.numeric)) |> 
+  filter(!is.na(above_dpd_hrl_fp_rear)) |> 
+  pivot_longer(below_dpd_hrl_ic_rear:above_dpd_hrl_ic_rear,
+               names_to = "reach_ic",
+               values_to = "ic_suitable_acres") |> 
+  mutate(reach_ic = ifelse(str_detect(reach_ic, "below"), "below_dpd", "above_dpd")) |> 
+  pivot_longer(below_dpd_hrl_fp_rear:above_dpd_hrl_fp_rear,
+               names_to = "reach_fp",
+               values_to = "fp_suitable_acres") |> 
+  mutate(reach_fp = ifelse(str_detect(reach_ic, "below"), "below_dpd", "above_dpd"),
+         reach = ifelse(reach_fp == reach_ic, reach_fp, NA)) |> 
+  select(flow_cfs, reach, ic_suitable_acres, fp_suitable_acres)
+
+# combine across reaches
+upper_reach_length = 12.7
+lower_reach_length = 11.4
+tot_study_len = upper_reach_length + lower_reach_length
+
+# combine two reaches
+yuba_hrl_combined <- yuba_hrl |> 
+  mutate(reach_length = ifelse(reach == "below_dpd", lower_reach_length, upper_reach_length)) |> 
+  group_by(flow_cfs) |> 
+  summarise(ic_hrl_acres = sum(ic_suitable_acres/reach_length * reach_length / tot_study_len, na.rm = T),
+            fp_hrl_acres = sum(fp_suitable_acres/reach_length * reach_length / tot_study_len, na.rm = T)) |> 
+  ungroup() |> 
+  glimpse()
+
+knitr::kable(head(yuba_hrl_combined))
+```
+
+### Plot comparison
```

Comment:
Would be good to have same habitat types showing on same plot. Also yuba HRL does not have a very large range of flows...we don't need to do anything about that right now but would be good to think about